crates/typst/src/model/heading.rs

```diff
@@ -163,6 +165,21 @@ pub struct HeadingElem {
     #[default(Smart::Auto)]
     pub bookmarked: Smart<bool>,
 
+    /// The indent all but the first line of a heading should have.
+    ///
+    /// The default value of `{auto}` indicates that the subsequent heading
+    /// lines will be indented based on the width of the numbering.
+    ///
+    /// Custom lengths will indent the lines at least as much as the
+    /// numbering.
```

Comment:
This strikes me as a bit strange. For a hypothetical `body-indent` it would have made sense not to overlap, but for a hanging indent, I'm not sure it makes sense.

Reply:
Oh yes, of course. Seems I confused something here ^^'